native/src/rapidsml_jni.hpp

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <raft/linalg/cublas_wrappers.h>
+
+namespace {
```

Comment:
[Anonymous namespaces in C++ headers](https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file#:~:text=an%20%23include%20directive.-,Do%20not%20define%20an%20unnamed%20namespace%20in%20a%20header%20file,used%20within%20that%20translation%20unit.) is an anti-pattern.

Reply:
Removed, and url added to bookmark 👍